hackathon_site/hardware/serializers.py

```diff
@@ -39,6 +39,14 @@ def get_unique_hardware_count(obj: Category) -> int:
         return obj.hardware_set.annotate(Count("id", distinct=True)).count()
 
 
+class ChangeOrderStatusSerializer(serializers.ModelSerializer):
```

Comment:
Nit, but for consistency with the others can you call this `OrderChangeSerializer`?